central/complianceoperator/v2/suites/datastore/datastore.go

```diff
@@ -0,0 +1,52 @@
+package datastore
+
+import (
+	"context"
+	"testing"
+
+	pgStore "github.com/stackrox/rox/central/complianceoperator/v2/suites/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+)
+
+// DataStore is the entry point for storing/retrieving compliance operator suite.
+//
+//go:generate mockgen-wrapper
+type DataStore interface {
+	// GetSuite returns the suite for the given id
+	GetSuite(ctx context.Context, id string) (*storage.ComplianceOperatorSuite, bool, error)
+
+	// GetSuitesByCluster retrieve suites by the cluster
+	GetSuitesByCluster(ctx context.Context, clusterID string) ([]*storage.ComplianceOperatorSuite, error)
+
+	// UpsertSuite adds the suite to the database
+	UpsertSuite(ctx context.Context, suite *storage.ComplianceOperatorSuite) error
+
+	// UpsertSuites adds the suites to the database
```

Comment:
nit: Is this one being used?  If not, we can remove it.

Reply:
This is used for testing for the convenience of testing now.

Reply:
This is at least used in the tests.